src/util/snapshots.tsx

```diff
@@ -126,7 +109,9 @@ export const getSnapshotSchema = (
       .test(
         ...testDuplicateSnapshotName(instance, controllerState, snapshotName),
       )
-      .test(...testForbiddenChars()),
+      .matches(/^[A-Za-z0-9-_.:]+$/, {
+        message: "Only alphanumeric and _ . - : characters allowed",
```

Comment:
Can we make the message more readable and specific for the reader:

```suggestion
        message: "Only alphanumeric and _ . - : characters allowed",
```